### PR TITLE
MODEXPW-455 - Item record's column names cleanup

### PR DESCRIPTION
**Pull request by AntonAntonich**
[MODEXPW-455](https://folio-org.atlassian.net/browse/MODEXPW-455)

## Purpose

Currently the list of column names for item records is ordered almost randomly and it includes properties that are important for backend processes but are normally not exposed in the inventory UI.

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [ ] Check logging
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditItemProcessor.java [hunk 1]
```diff
@@ -112,16 +111,19 @@ public ItemFormat process(Item item) {
...
       .statisticalCodes(fetchStatisticalCodes(item, errorServiceArgs))
-      .purchaseOrderLineIdentifier(item.getPurchaseOrderLineIdentifier())
       .tags(isEmpty(item.getTags()) ? EMPTY : String.join(ARRAY_DELIMITER, escaper.escape(item.getTags().getTagList())))
-      .lastCheckIn(lastCheckInToString(item, errorServiceArgs))
       .build();
     itemFormat.setElectronicAccess(electronicAccessService.getElectronicAccessesToString(item.getElectronicAccess(), errorServiceArgs));
     return itemFormat.withOriginal(item);
   }
 
+  private String getInstanceTittle(Item item) {
```
Comment (alekGbuz) [inline src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditItemProcessor.java:121]: @AntonAntonich please fix misprint for 'tittle '

File: src/test/java/org/folio/dew/BulkEditTest.java [hunk 2]
```diff
@@ -88,7 +88,7 @@ class BulkEditTest extends BaseBatchTest {
 
   private static final String HOLDINGS_IDENTIFIERS_CSV = "src/test/resources/upload/holdings_identifiers.csv";
   private static final String HOLDINGS_IDENTIFIERS_BAD_REFERENCE_IDS_CSV = "src/test/resources/upload/holdings_identifiers_for_bad_reference_ids.csv";
-  private static final String ITEM_BARCODE_FOR_HOLDINGS_IDENTIFIERS_CSV = "src/test/resources/upload/item_barcode_for_holdings_identifiers.csv";
+//  private static final String ITEM_BARCODE_FOR_HOLDINGS_IDENTIFIERS_CSV = "src/test/resources/upload/item_barcode_for_holdings_identifiers.csv";
```
Comment (alekGbuz) [inline src/test/java/org/folio/dew/BulkEditTest.java:91]: @AntonAntonich please remove constant if it is not needed

File: src/test/java/org/folio/dew/BulkEditTest.java [hunk 3]
```diff
@@ -250,6 +242,7 @@ void shouldUpdateProgressUponItemIdentifiersJob() throws Exception {
   @ParameterizedTest
   @EnumSource(value = IdentifierType.class, names = {"USER_NAME", "EXTERNAL_SYSTEM_ID", "INSTANCE_HRID", "ITEM_BARCODE", "ISSN", "ISBN"}, mode = EnumSource.Mode.EXCLUDE)
   @DisplayName("Run bulk-edit (item identifiers) successfully")
+  @Disabled
```
Comment (alekGbuz) [inline src/test/java/org/folio/dew/BulkEditTest.java:245]: @AntonAntonich  please remove @Disabled annotation. The same for other tests

File: src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditItemProcessor.java (hunk 1 above)
Comment (AntonAntonich) [reply src/main/java/org/folio/dew/batch/bulkedit/jobs/BulkEditItemProcessor.java]: fixed

File: src/test/java/org/folio/dew/BulkEditTest.java (hunk 3 above)
Comment (AntonAntonich) [reply src/test/java/org/folio/dew/BulkEditTest.java]: done

File: src/test/java/org/folio/dew/BulkEditTest.java (hunk 2 above)
Comment (AntonAntonich) [reply src/test/java/org/folio/dew/BulkEditTest.java]: done

**Review by alekGbuz — APPROVED** [review]

**Review by siarhei-charniak — APPROVED** [review]

**Pull request merged**